benchmarking/README.md

```diff
@@ -8,6 +8,13 @@ This is work in progress
 
```

Comment:
In commit "test/statistics-analysis: Fix typo and readme file", the commit message and subject now look a bit weird as it is combined from two different commits.
Could you change this to
```
test/statistics-analysis: Fix topology_aware name in scripts

Fix the typo in run_tests.sh, the topology aware variable
was incorrectly written.
Fix the name of the topology-aware jaeger file, it was
topology-aware but file name is currently topology_aware

Add python dependency list to readme file.
```